deploy/osps/default/osp-amzn2.yaml

```diff
@@ -458,6 +459,12 @@ spec:
               systemctl restart systemd-modules-load.service
               sysctl --system
 
+              # Override hostname if /etc/machine-name exists
+              if [ -x "$(command -v hostnamectl)" ] && [ -s /etc/machine-name ]; then
```

Comment:
Probably this is not gonna work, as it is not possible to override the hostname for aws machines

Reply:
In the case of AWS, we never set `/etc/machine-name` so this is noop https://github.com/ahmedwaleedmalik/operating-system-manager/blob/osp-fixes/pkg/generator/generator.go#L227. Let me remove this.